frontend/public/components/build-config.tsx

```diff
@@ -186,41 +153,80 @@ const allStrategies = [
   BuildStrategyType.Source,
   BuildStrategyType.Custom,
 ];
-const filters = [
-  {
-    filterGroupName: 'Build Strategy',
-    type: 'build-strategy',
-    reducer: buildStrategy,
-    items: _.map(allStrategies, (strategy) => ({
-      id: strategy,
-      title: strategy,
-    })),
-  },
-];
 
-export const BuildConfigsList: React.SFC = (props) => (
-  <Table
-    {...props}
-    aria-label="Build Configs"
-    Header={BuildConfigsTableHeader}
-    Row={BuildConfigsTableRow}
-    virtualize
-  />
-);
+export const BuildConfigsList: React.SFC = (props) => {
+  const { t } = useTranslation();
+  const BuildConfigsTableHeader = () => {
+    return [
+      {
+        title: t('build-config~Name'),
+        sortField: 'metadata.name',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[0] },
+      },
+      {
+        title: t('build-config~Namespace'),
+        sortField: 'metadata.namespace',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[1] },
+        id: 'namespace',
+      },
+      {
+        title: t('build-config~Labels'),
+        sortField: 'metadata.labels',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[2] },
+      },
+      {
+        title: t('build-config~Created'),
+        sortField: 'metadata.creationTimestamp',
+        transforms: [sortable],
+        props: { className: tableColumnClasses[3] },
+      },
+      {
+        title: '',
+        props: { className: tableColumnClasses[4] },
+      },
+    ];
+  };
+  return (
+    <Table
+      {...props}
+      aria-label={t('build-config~Build Configs')}
+      Header={BuildConfigsTableHeader}
+      Row={BuildConfigsTableRow}
+      virtualize
+    />
+  );
+};
 
 BuildConfigsList.displayName = 'BuildConfigsList';
 
-export const BuildConfigsPage: React.SFC<BuildConfigsPageProps> = (props) => (
-  <ListPage
-    {...props}
-    title="Build Configs"
-    kind={BuildConfigsReference}
-    ListComponent={BuildConfigsList}
-    canCreate={true}
-    filterLabel={props.filterLabel}
-    rowFilters={filters}
-  />
-);
+export const BuildConfigsPage: React.FC<BuildConfigsPageProps> = (props) => {
+  const { t } = useTranslation();
+  const filters = [
+    {
+      filterGroupName: t('build-config~Build Strategy'),
```

Comment:
As part of this effort, we're updating the capitalization to the guidelines here:

http://openshift.github.io/openshift-origin-design/conventions/documentation/capitalization.html

So this should be `Build strategy`.

```suggestion
      filterGroupName: t('build-config~Build strategy'),
```